stable_baselines3/common/logger.py

```diff
@@ -21,18 +27,21 @@ class KVWriter(object):
     Key Value writer
     """
 
-    def writekvs(self, kvs: Dict) -> None:
+    def write_key_values(self, key_values: Dict, key_excluded: Dict, step: int) -> None:
```

Comment:
Can't we merge `writekvs` and `writeseq`? 
so we can simply name it `write()`

Reply:
You mean eliminating SeqWriter class and have a single function?
```python
def write(self, key_values: Dict, key_excluded: Dict, sequence: List, step: int) -> None:
```

Reply:
Imaybe not...
or rename to `write()`, `write_sequence()`
to be honest, I did not really take the time to read and understand what does `ẁrite_seq` exactly do, but the name is quite obscure...